osmosis-1/osmosis-1.assetlist.json

```diff
@@ -10909,7 +10909,7 @@
       "keywords": [
         "osmosis-main",
         "osmosis-frontier"
-      ]
+        } 
```

Comment:
this is the wrong kind of closing bracket (should be ']'

Reply:
Fixed